lib/parser/object-declaration-parser.ts

```diff
@@ -52,36 +52,46 @@ export class ObjectDeclarationParser extends ParserBase {
 
     } while (this.getToken().getLText() === ',');
     this.expect(':');
+
     if (file) {
       const typeToken = this.consumeToken();
-      for (const file of objects as OFileVariable[]) {
+      for (const file of objects.slice(objects.length - 1) as OFileVariable[]) {
         const typeRead = new ORead(file, typeToken);
         file.typeReference = [typeRead];
+        let tokens, endToken;
         if (this.maybe('open')) {
-          const [tokens] = this.advanceParenthesisAware(['is', ';'], true, false);
+          [tokens, endToken] = this.advanceParenthesisAware(['is', ';', ...this.NotExpectedDelimiter], true, false);
           file.openKind = new ExpressionParser(this.state, file, tokens).parse();
         }
         if (this.maybe('is')) {
-          const [tokens] = this.advanceParenthesisAware([';'], true, false);
+          [tokens, endToken] = this.advanceParenthesisAware([';', ...this.NotExpectedDelimiter], true, false);
           file.logicalName = new ExpressionParser(this.state, file, tokens).parse();
         }
+        if (!endToken) {
+          endToken = this.getToken();
+        }
+        if (this.NotExpectedDelimiter.includes(endToken.getLText())) {
+          this.state.messages.push({
+            message: `Unexpected ${endToken.text} in object declaration. Assuming forgotten delimiter`,
+            range: endToken.range
+          });
```

Comment:
I would just say missing ';' and add a quick fix to add the semicolon. `is` isn't really a delimiter, it is just allowed because a file variable allows a `file_open_information` which is optional. Therefore, the declaration would also be valid (if sometimes maybe not correct) with just a semicolon.